### DISCUSSION TITLE
2305.40000.2.0 Not Found.

### DISCUSSION THREAD

**Discussion by 7gxycn08** [LSPosed/MagiskOnWSALocal#575] [Q&A]
They just released version 2305.40000.2.0 can't find it In MagiskOnWSALocal max found is 2304.

**Comment by williamvyong** [comment]
still need to wait a few days i think

**Comment by Almighty188** [comment]
MagiskOnWSALocal only will use the same versions as released on Microsoft Store using this method:

1) Open https://store.rg-adguard.net/ in browser
2) In the pull down menu, either  select "ProductId" and put 9P3395VX91NR in the search box 
*OR*
select “URL (link)” and put https://www.microsoft.com/en-us/p/windows-subsystem-for-android/9p3395vx91nr in the search box
3) Next, select the channel as "fast" from the second drop-down menu and then click on the check mark
4) Look for the file named MicrosoftCorporationII.WindowsSubsystemForAndroid_*_neutral_~_8wekyb3d8bbwe.msixbundle where * is the version number
For step 3, you can use slow, RP (Release Preview) and Retail instead as this will be the exact same version as what's in Microsoft Store and also the latest version that MagiskOnWSALocal will use.

but not the Preview versions.   See the General Discussions area for posts about new releases and if not on MagiskOnWSALocal, it will contain instructions on what to do to get the version in question and working.

**Reply by 7gxycn08** [reply]
> MagiskOnWSALocal only will use the same versions as released on Microsoft Store using this method:
> 
>     1. Open https://store.rg-adguard.net/ in browser
> 
>     2. In the pull down menu, either  select "ProductId" and put 9P3395VX91NR in the search box
>        _OR_
>        select “URL (link)” and put https://www.microsoft.com/en-us/p/windows-subsystem-for-android/9p3395vx91nr in the search box
> 
>     3. Next, select the channel as "fast" from the second drop-down menu and then click on the check mark
> 
>     4. Look for the file named MicrosoftCorporationII.WindowsSubsystemForAndroid_*_neutral_~_8wekyb3d8bbwe.msixbundle where * is the version number
>        For step 3, you can use slow, RP (Release Preview) and Retail instead as this will be the exact same version as what's in Microsoft Store and also the latest version that MagiskOnWSALocal will use.
> 
> 
> but not the Preview versions. See the General Discussions area for posts about new releases and if not on MagiskOnWSALocal, it will contain instructions on what to do to get the version in question and working.

its not on https://store.rg-adguard.net/

**Reply by Almighty188** [reply]
> > MagiskOnWSALocal only will use the same versions as released on Microsoft Store using this method:
> > ```
> > 1. Open https://store.rg-adguard.net/ in browser
> > 
> > 2. In the pull down menu, either  select "ProductId" and put 9P3395VX91NR in the search box
> >    _OR_
> >    select “URL (link)” and put https://www.microsoft.com/en-us/p/windows-subsystem-for-android/9p3395vx91nr in the search box
> > 
> > 3. Next, select the channel as "fast" from the second drop-down menu and then click on the check mark
> > 
> > 4. Look for the file named MicrosoftCorporationII.WindowsSubsystemForAndroid_*_neutral_~_8wekyb3d8bbwe.msixbundle where * is the version number
> >    For step 3, you can use slow, RP (Release Preview) and Retail instead as this will be the exact same version as what's in Microsoft Store and also the latest version that MagiskOnWSALocal will use.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > but not the Preview versions. See the General Discussions area for posts about new releases and if not on MagiskOnWSALocal, it will contain instructions on what to do to get the version in question and working.
> 
> its not on https://store.rg-adguard.net/

DId you bother to read what I said.  I said it will only show the latest as what's available on Microsoft Store and MagiskOnLocal and in this case, the latest is 2304.40000.7.0 for fast/slow/release preview/retail channels from Microsoft Store which is what store.rg-adguard.net downloads.  2305.40000.2.0 is a preview version which like all previous preview versions is not available there and I already told you if it's not there, to read the general discussions and look for the version you need being announced which I have been posting which tells you where to get it and how to get it working....  Latest on Microsoft Store does not mean the latest released version, it is just he latest version Microsoft makes available there, I can count 4 versions that were preview  2303.40000.2.0, 2304.40000.3.0, 2304.40000.4.0, 2305.40000.2.0 that did not make it there in the last 3 months alone, not sure if it's because someone at Microsoft forgot to upload it to Microsoft Store or not.

![image](https://github.com/LSPosed/MagiskOnWSALocal/assets/78993393/082eddf0-40a3-4a36-8793-a0d0bd6b6962)
and to prove to you that I have MagiskOnWSALocal with WSA 2305.40000.2.0, see this which would have taken you less than 5 minutes if you actually read what I wrote earlier as all you really needed to do is download the file, rename it and place it in the downloads folder:
![image](https://github.com/LSPosed/MagiskOnWSALocal/assets/78993393/5959d2bd-3d80-4620-a099-c878dcdc3e6c)
 

Read the following thread too on the official WSA discussions as I asked the same question why some versions are not on Microsoft Store:
https://github.com/microsoft/WSA/discussions/303

**Reply by 7gxycn08** [reply]
All I see is it's because someone at Microsoft forgot to upload it to Microsoft Store. ok.

**Reply by Almighty188** [reply]
> All I see is it's because someone at Microsoft forgot to upload it to Microsoft Store. ok.

Would it matter since what matters is you find a way to get it and get it working.  The results are what matters, not how you get there.  No different than Windows Terminal has a new update, people have been able to either download it from Github or use Winget but Microsoft Store is still the previous version.  You have a choice, either take it using a alternative method or wait for it to show up which may never happen as I had mentioned with the 4 versions.  It's a solution, not something up for negotiation.   If you want to complain, complain to Microsoft and they might throw you a bone if you are lucky.   Remember also you said you can't find it and a solution has been given on where to find it and download it.  It's a free world, do whatever you want but someone who does not even follow basic netiquette.
https://coursedesign.colostate.edu/obj/corerulesnet.html

**Reply by 7gxycn08** [reply]
I don't see a solution just you typing bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla.

**Reply by Almighty188** [reply]
> I don't see a solution just you typing bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla.

The solution is there, read the posts I pointed you to. 
![image](https://github.com/LSPosed/MagiskOnWSALocal/assets/78993393/d0a58a93-eaf4-484f-bcb2-5aee3be07084)

Already told you where exactly to download it.  If you don't want to read, then it's not worth my time to even respond now or ever again.  Try to be nice by helping and all I get is no appreciation whatsoever, someone posts with a one liner and trying to start a flame war just like a troll.  And I am sorry that you can't read.

**Reply by 7gxycn08** [reply]
Ok thanks.
